src/test/java/com/snowflake/kafka/connector/internal/SinkServiceIT.java

```diff
@@ -763,14 +763,14 @@ public void testCleanerRecover() throws Exception
 
     System.out.println("break connection");
     doThrow(SnowflakeErrors.ERROR_2001.getException()).when(spyConn).purgeStage(anyString(), anyList());
-    // Sleep two minutes so that cleaner encounters two exceptions.
-    Thread.sleep(SnowflakeSinkServiceV1.CLEAN_TIME * 2);
+    // Sleep 6 minutes so that cleaner encounters 6 exceptions. Just to make sure cleaner restart is triggered
+    Thread.sleep(6 * 60 * 1000);
```

Comment:
Why are you hardcoding this instead of using CLEAN_TIME

Reply:
CLEAN_TIME is currently set to 1 minutes. I found the sleep time here this is not really associated with CLEAN_TIME. We just want to sleep long enough so that mocked broken api is triggered and the total sleep time should be 10 minutes.